src/features/visualizations/migration/pattern/Migration.js

```diff
@@ -134,8 +134,9 @@ export default withErrorBoundary(() => {
         have become less predictable with the advent of increasing climate
         change, so too have the patterns of bird migrations become more varied.
         Bird observations during the peak of the season have decreased between
-        1970 and today, suggesting that the peak of the season is now earlier
-        and that the populations of various species may be decreasing.
+        1970 and today, suggesting that the peak of the season is now earlier.
+        Similar data suggests that the populations of various species may be
+        decreasing.
```

Comment:
My final act of butting in is merely to request that the full copy for the migration visualization be included, which just means adding this paragraph:

"A density plot allows us to check the distribution of a numeric variable. This is useful when examining a new dataset. A ridgeline plot such as this one lets us look at the distribution of bird observations amongst several groups. In this case, each group is the migration for that year. Each ‘observation’ is the number of sightings for a particular species."

I'll see myself out now.